### DISCUSSION TITLE
Firefox does not work correctly

### DISCUSSION THREAD

**Discussion by albert0346** [Budibase/budibase#11429] [Help] [answered]
Hi,

I created a table in the Firefox browser. When I hover over the column title the tree dots appear on the right side of the column. But when I click on the dots nothing happens. But this works with Chrome!

Is Firefox NOT the browser which works correctly with Budibase?

Thanks in advance for your help.

Albert

**Comment by ConorWebb96** [comment]
Hey @albert0346,

Could you provide some screenshots so I can better understand what you are seeing?

A lot of the developers use Firefox as their main developing browser. Firefox should work like the other browsers, if it is a bug and we can replicate it. We will get it fixed ASAP.

Thanks ahead of time for the additional information.

**Reply by albert0346** [reply]
Here is a screenshot with the 3 dots circled red. Also with the Chrome browser 
![test](https://github.com/Budibase/budibase/assets/3371780/9e1b4d3d-bbbe-4d8f-82a2-6c8dd0b7d78d)
I had to click twice, but at least It worked then. But as I wrote it does not work with Firefox.

**Reply by ConorWebb96** [reply]
Hey @albert0346,

I'm not seeing any issue with this for Firefox on my end? This seems to be display properly/as expected for me on this browser.
What version of Firefox are you using? (I've attached a screenshot of my Firefox showing it correctly.)
<img width="943" alt="Screenshot 2023-08-03 at 12 12 38" src="https://github.com/Budibase/budibase/assets/126772285/c59dabf3-5149-47d6-b093-29bdb9847272">

Many thanks.

**Reply by albert0346** [reply]
I have Version 115.0.3 (64-Bit) on Windows 10.

**Reply by ConorWebb96** [reply]
Hey @albert0346,

I've just tried to replicate your issue on Firefox 115.0.3 on my windows machine and it still seems to be working perfectly fine. I'm not seeing any double click issue that you have described occurring?

Can you check your console to see if any errors are being throw when this strange behaviour is occurring?

**Reply by albert0346** [reply]
You see, I'm a simple user, not a programmer. How can I check the Firefox console?

**Reply by albert0346** [reply]
I found it. What should I look for?

**Reply by albert0346** [reply]
I have here something from the console. I don't know if it is related to the mentioned error. There are more of those messages.

![error](https://github.com/Budibase/budibase/assets/3371780/3a6d2a5f-c0e9-4b1d-b38a-e333551af212)

**Reply by ConorWebb96** [reply]
Hey @albert0346,

Can you open the uncaught error? and take a screenshot of what it says. (You just need to click the > arrow beside the word object.) 

- Does this only show up when you click the 3 dots or when the page is loaded? 
- Does this only happen within this app? 
- Can you reproduce this across all your apps?
- Are you self-hosted or cloud?
- What version of Budibase are you using?

**Reply by albert0346** [reply]
I clicked again the 3 dots and looked on the console, but now I did not find any error messages.
I have only the Test app.
I am on the cloud.
I use version 2.8.30.

Can you not look into my account and see what's going on?

**Comment by JCV67** [comment]
Also, in Firefox, table columns order is in reverse.

**Comment by melohagan** [comment, marked as answer]
Hey guys @albert0346 @JCV67 @ConorWebb96 , we're aware of the issue: https://github.com/Budibase/budibase/issues/10689
